.github/workflows/Continuous-integration.yml

```diff
@@ -0,0 +1,65 @@
+name: LPS22DF Continuous Integration
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*'
+      - '**.md'
+      - '**.txt'
+  pull_request:
+    paths-ignore:
+      - '*'
+      - '**.md'
+      - '**.txt'
+jobs:
+  astyle_check:
+    runs-on: ubuntu-latest
+    name: AStyle check
+    steps:
+    # First of all, clone the repo using the checkout action.
+    - name: Checkout
+      uses: actions/checkout@main
+
+    - name: Astyle check
+      id: Astyle
+      uses: stm32duino/actions/astyle-check@main
+
+    # Use the output from the `Astyle` step
+    - name: Astyle Errors
+      if: failure()
+      run: |
+        cat ${{ steps.Astyle.outputs.astyle-result }}
+        exit 1
+  spell-check:
```

Comment:
As stated by the action annotation (see https://github.com/stm32duino/LPS22DF/actions/runs/2622832799), spell check is deprecated.
Please update to use CodeSpell instead. Ex: https://github.com/stm32duino/STM32SD/blob/main/.github/workflows/codespell.yml